modules/dcache/src/main/java/org/dcache/pool/classic/IoQueueManager.java

```diff
@@ -451,16 +461,43 @@ public Serializable call() throws NoSuchElementException
                     comparator = comparator.reversed();
                 }
 
+                // Prepare filters
+                Predicate<MoverRequestScheduler.PrioritizedRequest> clientIpFilter;
+                if (clientHost != null && !clientHost.isEmpty()) {
+                    try {
+                        // check and resolve host
```

Comment:
This is a non-blocking comment (as it's mostly stylistic), but these comments don't actually help.  The code already describes what it does, so the comments don't add anything.

Instead, they're potential problems in the future -- when someone refactors this code, if they forget to update/move the comments then there's a potential inconsistency between what the code does and what the comment says it should be doing.  This makes debugging harder.

Comments are reasonable to describe *why* a particular behaviour is important, rather than describing what the code is doing -- if you find need a comment to describe what the code is doing, it's better to rewrite the code! :smile: